main.qml

```diff
@@ -1000,13 +969,13 @@ ApplicationWindow {
             informationPopup.onCloseCallback = null;
             informationPopup.open();
         } else {
-            if (transactionDescription.length > 0) {
-                for (var i = 0; i < txid.length; ++i)
-                  currentWallet.setUserNote(txid[i], transactionDescription);
+            if (txConfirmationPopup.transactionDescription.length > 0) {
+                currentWallet.setUserNote(txid, txConfirmationPopup.transactionDescription);
```

Comment:
`txid` is a list

Reply:
`txid` seems to always return a single txid, so I decided to simplify it. Is there any situation with multiple txid's?

Reply:
Please don't simplify it

Reply:
The interface (`transactionCommitted` signal) returns a list and it should be treated as a list.